src/main/java/com/donateblood/com/donateblood/controller/DonateBloodController.java

```diff
@@ -15,10 +16,33 @@ public class DonateBloodController {
 	
 	@Autowired
 	ExternalApiServie apiServie;
+
+	@Autowired
+	BloodGroupsService bloodGroupsService;
+
+	@Autowired
+	UserRegistrationService userRegistrationService;
+
 	
 	@GetMapping("/getCountries")
-	public List<Object> getCountries() {
+	public List<String> getCountries() {
 		return apiServie.getCountries();
 	}
```

Comment:
Please make return type as response entity with Http status code
apply this for all end point

Reply:
Modified